game-core/src/main/java/games/strategy/triplea/ai/weak/DoesNothingAi.java

```diff
@@ -41,23 +41,18 @@ protected void purchase(
       new WeakAi(this.getName())
           .purchase(purchaseForBid, pusToSpend, purchaseDelegate, data, player);
     }
-    pause();
   }
 
   @Override
   protected void tech(
-      final ITechDelegate techDelegate, final GameData data, final GamePlayer player) {
-    pause();
```

Comment:
The AI never researches tech? I'm just guessing, is there another reason to remove the tech delegate pause here? I'm also curious about the politics action pause removal as well

Reply:
This is the donorhingai

Reply:
Hehe, took a moment to decode "donorhingai", kept reading that as "donor" "hingai"

Cool, thanks for pointing that out, makes sense :+1: